micro_ros_setup/config/nuttx/olimex-stm32-e407/build.sh

```diff
@@ -0,0 +1,15 @@
+NUTTX_DIR=$FW_TARGETDIR/NuttX
```

Comment:
Again, what's the rationale for making this script board-specific instead of generic?